blueocean-core-js/src/js/fetch.js

```diff
@@ -290,7 +290,10 @@ function getPrefetchedDataFuture(url) {
                 // the prefetched data as the value.
                 if (trimRestUrl(preFetchEntry.restUrl) === trimmedUrl) {
                     try {
-                        return Promise.resolve(JSON.parse(preFetchEntry.data));
+                        const parse = JSON.parse(preFetchEntry.data);
+                        return Promise.resolve(parse);
+                    } catch (e) {
+                        console.error('preFetch Error', e);
```

Comment:
@cliffmeyers I will have a look. However the prefetch seems not the root cause. Still trying to pin down where the problems happens. FF were throwing problems in parsing the sse-event so maybe it is some faulty events. Still debugging some more.